src/event/event.schema.js

```diff
@@ -0,0 +1,14 @@
+import { schema } from 'normalizr';
+// import omit from 'lodash/omit';
+
+import { userSchema } from '../user/user.schema';
+import { repoSchema } from '../repository/repository.schema';
+
+export const eventSchema = new schema.Entity('events', {
+  actor: userSchema,
+  repo: repoSchema,
+  payload: {
+    forkee: repoSchema,
+    member: userSchema,
+  },
+});
```

Comment:
This is one of my favorite files :) 
We describe the event structure and link nested objects to their own schema